spec/models/partners/profile_spec.rb

```diff
@@ -215,4 +215,19 @@
   describe "versioning" do
     it { is_expected.to be_versioned }
   end
+
+  describe "default values" do
```

Comment:
Not sure this needs its own test. You're basically asserting the default database values.

Reply:
Removed.